sdcm/tester.py

```diff
@@ -3031,11 +3031,18 @@ def tearDown(self):
         self.log.info('Test ID: {}'.format(self.test_config.test_id()))
         self._check_alive_routines_and_report_them()
         self._check_if_db_log_time_consistency_looks_good()
+<<<<<<< HEAD
         self.remove_python_exit_hooks()
```

Comment:
@soyacz 

I think this call is exactly why we didn't see this issue in v16

we should still backport the fix, but withut taking this out